boards/arm/stm32g071b_disco/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# STM32G071B Discovery board configuration
+
+# Copyright 2021 The Chromium OS Authors
```

Comment:
Is it your copyright, or copy/paste leftover ?

Reply:
I'm a Google employee working on the Chrome OS team and thought I should use the same copyright as the board in google_kukui.